snippets/c.snippets

```diff
@@ -130,6 +130,84 @@ snippet fun
 	{
 		${4}
 	}
+# function definition with zero parameters
+snippet fun0
+	${1:void} ${2:function_name}()
+	{
+		${3}
```

Comment:
I'm keen to merge this in as it has been left for a while, this improvement can come in a later PR, thanks. :)